pandas/core/nanops.py

```diff
@@ -1241,7 +1241,7 @@ def nancorr(a, b, method="pearson", min_periods=None):
 
 
 def get_corr_func(method):
-    if method in ["kendall", "spearman"]:
+    if method in ["kendall", "spearman", "pearson"]:
```

Comment:
actrually, why is this necessary at all? this just uses numpy and not scipy

Reply:
You asked me to:)

ref - https://github.com/pandas-dev/pandas/pull/30461#discussion_r361339781

Reply:
ahh right, but this is not actually the correct check, instead do like
```
if method in ['kendall', 'spearman']:
   .....
elif method in ['pearson']:
    pass
elif callable(method):
   ....
else:
     raise ValueError("....in valid method passed')
```

Reply:
```python
raise ValueError(f"Unrecognized method '{method}'")
```

Sounds good? (probably not, my English is not so good)

And also, should I remove all the "Skip" decorators Iv'e added in https://github.com/pandas-dev/pandas/pull/30603/commits/ed6f9c951be18edfdd90a659ae3f833ba8dcbbe0 ?

Reply:
sure & yes remove the skips